content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -433,8 +433,8 @@ Jobオブジェクトは、Podの確実な並列実行をサポートするた
 並列計算にはいくつかのパターンがあり、それぞれに長所と短所があります。
 トレードオフの関係にあるのは:
 
-- 各作業項目に1つのJobオブジェクト vs. すべての作業項目に1つのJobオブジェクト。  
-　後者は大量の作業項目を処理する場合に適しています。  
```

Comment:
I think this change itself is ok, but this removal of trailing spaces is causing removing newlines (and the spaces between sentences are unexpected maybe)
Revert the spaces, or remove the leading spaces?

Reply:
Thank you!!

I fixed (ref. https://github.com/kubernetes/website/pull/44757/commits/ae341b6890af293c6ac861fe8990c842f60aa022)

I don't know why this happen...